### PR TITLE
Add Privileged property to ExecCreateCmd

### LINKED ISSUE #902: Add Privileged property to ExecCreateCmd
It is currently not possible to set the privileged property for exec commands:
https://docs.docker.com/engine/api/v1.29/#operation/ContainerExec

### PR DESCRIPTION
**Pull request by Jotschi**
Fixes #902

---
This change is [Reviewable](https://reviewable.io/reviews/docker-java/docker-java/903)

### CODE REVIEW

File: src/main/java/com/github/dockerjava/core/command/ExecCreateCmdImpl.java [hunk 1]
```diff
@@ -26,6 +26,9 @@
     @JsonProperty("Tty")
     private Boolean tty;
 
+    @JsonProperty("Privileged")
+    private Boolean isPrivileged;
```
Comment (KostyaSha) [inline src/main/java/com/github/dockerjava/core/command/ExecCreateCmdImpl.java]: just `privileged` without is.

File: src/main/java/com/github/dockerjava/core/command/ExecCreateCmdImpl.java [hunk 2]
```diff
@@ -109,6 +118,11 @@ public Boolean hasTtyEnabled() {
     }
 
     @Override
+    public Boolean isPrivileged() {
```
Comment (KostyaSha) [inline src/main/java/com/github/dockerjava/core/command/ExecCreateCmdImpl.java]: for objects it should be `getPrivileged()` because Boolean is object

**Comment by Jotschi** [comment]
@KostyaSha I have updated the PR accordingly

File: src/main/java/com/github/dockerjava/core/command/ExecCreateCmdImpl.java [hunk 3]
```diff
@@ -26,6 +26,9 @@
     @JsonProperty("Tty")
     private Boolean tty;
 
+    @JsonProperty("Privileged")
+    private Boolean privileged;
```
Comment (KostyaSha) [inline src/main/java/com/github/dockerjava/core/command/ExecCreateCmdImpl.java]: please add 
```
/**
 * @since 1.29
 **/
```
if it was added since 1.29..

**Comment by Jotschi** [comment]
@KostyaSha I added the since annotation. Sorry that it took so long.

**Pull request merged**